examples/starwars/ios/Flutter/flutter_export_environment.sh

```diff
@@ -1,11 +1,8 @@
 #!/bin/sh
 # This is a generated file; do not edit or check into version control.
-export "FLUTTER_ROOT=/Users/mjr/flutter"
-export "FLUTTER_APPLICATION_PATH=/Users/mjr/Documents/code/libraries/graphql-flutter/examples/starwars"
-export "FLUTTER_TARGET=/Users/mjr/Documents/code/libraries/graphql-flutter/examples/starwars/lib/main.dart"
+export "FLUTTER_ROOT=C:\tools\flutter"
+export "FLUTTER_APPLICATION_PATH=C:\Users\mw\projects\opensource\graphql-flutter\examples\starwars"
+export "FLUTTER_TARGET=lib\main.dart"
 export "FLUTTER_BUILD_DIR=build"
-export "SYMROOT=${SOURCE_ROOT}/../build/ios"
-export "FLUTTER_FRAMEWORK_DIR=/Users/mjr/flutter/bin/cache/artifacts/engine/ios"
-export "FLUTTER_BUILD_NAME=1.0.0"
-export "FLUTTER_BUILD_NUMBER=1"
-export "TRACK_WIDGET_CREATION=true"
+export "SYMROOT=${SOURCE_ROOT}/../build\ios"
+export "FLUTTER_FRAMEWORK_DIR=C:\tools\flutter\bin\cache\artifacts\engine\ios"
```

Comment:
Nit: we should ignore this file

Reply:
Yes, let me see if I can update gitignore